tests/ab/test_data.py

```diff
@@ -0,0 +1,24 @@
+"""
+Tests for the ab.data module
+"""
+import datetime as dt
+
+import pytest
+
+from ab.data import calc_gps_week
+
+
+@pytest.mark.parametrize(
+    "date, gps_week",
+    [
+        # Date, GPS week
+        (dt.date(1980, 1, 6), 0),
+        (dt.date(1985, 7, 25), 288),
```

Comment:
Både online GPS kalendere og gnsscal (og koden i main) giver 289 for 1985,7,25

Reply:
```suggestion
        (dt.date(1985, 7, 25), 289),
```